keras_cv/models/models_test.py

```diff
@@ -120,6 +121,39 @@ def _test_model_can_be_used_as_backbone(self, app, last_dim, args):
         model = keras.Model(inputs=inputs, outputs=[backbone_output])
         model.compile()
 
+    def _test_model_convergence(self, app, args, accuracy):
+        tf.random.set_seed(1337)
+
+        train_ds = tfds.load(
+            "mnist",
+            # We load the test DS for convergence tests because it's smaler
+            # (10k records vs 60k in train)
+            split=["test"],
+            as_supervised=True,
+        )[0].batch(32)
+
+        @tf.function
+        def preprocess(image, label):
+            return image, tf.one_hot(label, 10)
+
+        train_ds = train_ds.map(preprocess)
+
+        with tf.distribute.MirroredStrategy().scope():
+            model = app(
+                include_top=True,
+                classes=10,
+                include_rescaling=True,
+                input_shape=(28, 28, 1),
+                **args
+            )
+            model.compile(
+                optimizer="adam", loss="categorical_crossentropy", metrics=["accuracy"]
+            )
+
+        results = model.fit(train_ds)
+
+        self.assertAlmostEqual(results.history["accuracy"][0], accuracy)
```

Comment:
I think this should be assertGreaterThan and we should provide a floor for convergence, not a guarantee

Reply:
Ack -- I've updated this to be a greater than assertion